plugins/GSdx/GSRendererHW.cpp

```diff
@@ -186,6 +186,42 @@ void GSRendererHW::SetGameCRC(uint32 crc, int options)
 	GSRenderer::SetGameCRC(crc, options);
 
 	m_hacks.SetGameCRC(m_game);
+
+	// Code for Automatic Mipmapping. Relies on game CRCs.
+	if (theApp.GetConfigI("mipmap_hw") == -1) {
+		switch (m_game.title) {
```

Comment:
I was wondering if a dictionary might be better compared to switch-case, but maybe not.